### DISCUSSION TITLE
A graphic instance, how to judge whether it is in the rectangle, how to change the color

### DISCUSSION THREAD

**Discussion by pcbren1** [pixijs/pixijs#7575] [Q&A]
The mouse coordinate position is 100，250, just inside the second rectangle
(in order to pursue efficiency, use one graphic instead of instantiating multiple graphics.) here are two questions?
1. How to judge whether the mouse position is in the second rectangle,
2. How to change the color of the second rectangle to green.
![89](https://user-images.githubusercontent.com/86771584/124341678-97c77f80-dbf0-11eb-8439-6d6afd26723e.png)

**Comment by eXponenta** [comment]
See:
https://github.com/pixijs/pixijs/discussions/7577

Or you can use color palette, how i do in my render for huge SVG. Of course this not in core and require a lot of knowledge in pixi + webgl

https://github.com/eXponenta/pixi5-svg/tree/pallete

Demo:
https://exponenta.games/games/map/

SVG that change a colors for specific regions without rebuilding.
